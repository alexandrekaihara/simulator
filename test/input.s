; MUST INPUT 10
INPUT AUX
LOAD AUX
SUB TEN
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO:
OUTPUT ONE
STOP
AUX: SPACE
TEN: CONST 10
MINUSONE: CONST -1
ONE: CONST 1
