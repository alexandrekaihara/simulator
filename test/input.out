12 14 10 14 2 15 8 11 13 16 14 13 17 14 0 10 -1 1
